full-moon/tests/cases/pass/fractional-numbers/source.lua

```diff
@@ -0,0 +1,4 @@
+local num = 0.5
+local num2 = 0.5e5
+local num3 = .5
+local num4 = .5e5
```

Comment:
Can you include a test for stuff like `1e-5` too? We've had trouble with that.

Reply:
That was added as a separate test under the /exponents/ folder, unless you want to merge them all into one big test?

Reply:
I didn't see that, nevermind.